packages/teleterm/src/services/pty/ptyHost/ptyProcess.ts

```diff
@@ -0,0 +1,57 @@
+import { PtyProcessType } from 'teleterm/sharedProcess/ptyHost';
+import { PtyHostClient } from '../types';
+
+export function createPtyProcess(
```

Comment:
It's a bummer that we're not able to return a class here. Each place which uses the return value of this function has to import the interface type `PtyProcessType`, rather than the class that actually implements the given interface.

I guess we can live with it. But it's so much easier is to inspect all the places where `PtyProcess` from `sharedProcess/ptyHost/ptyProcess.ts` is used vs the return value of this function. ;(

Reply:
True :(